Add-ons/UmbracoForms/Developer/Rendering-Scripts/index.md

```diff
@@ -5,26 +5,59 @@ versionTo: 10.0.0
 
 # Rendering Forms Scripts
 
-Forms output some JavaScript which is by default rendered right below the markup.
+Script delivery is done automatically using [Themes](/documentation/Add-ons/UmbracoForms/Developer/Themes/) without the need to include any additional code into your own views. The 'default' theme (bundled with Umbraco Forms) comes with the file `Scripts.cshtml` containing all the relevant logic & data required by the Umbraco Forms JavaScript. For reference, the file looks like this *(as per Umbraco Forms 10.2)*:
```

Comment:
This is great to add, but I think we should remove the "For reference... " sentence, and the code block.  It'll only get out of date here.  I'd rather direct people to where they can download the theme files.